src/Dfc.CourseDirectory.Web/Views/AddCourse/AddCourseDetails.cshtml

```diff
@@ -37,42 +37,16 @@
 
 
 @{
-    var newVenueObject = Context.Session.GetObject<AddedVenueModel>("NewAddedVenue");
-    if (newVenueObject != null)
-    {
-        string message = "";
-        if (!string.IsNullOrWhiteSpace(newVenueObject.VenueName))
-        {
-            message += string.Concat(newVenueObject.VenueName, ", ");
-        }
-        if (!string.IsNullOrWhiteSpace(newVenueObject.AddressLine1))
-        {
-            message += string.Concat(newVenueObject.AddressLine1, ", ");
-        }
-        if (!string.IsNullOrWhiteSpace(newVenueObject.AddressLine2))
-        {
-            message += string.Concat(newVenueObject.AddressLine2, ", ");
-        }
-        if (!string.IsNullOrWhiteSpace(newVenueObject.Town))
-        {
-            message += string.Concat(newVenueObject.Town, ", ");
-        }
-        if (!string.IsNullOrWhiteSpace(newVenueObject.County))
-        {
-            message += string.Concat(@newVenueObject.County, ", ");
-        }
-        if (!string.IsNullOrWhiteSpace(newVenueObject.PostCode))
-        {
-            message += newVenueObject.PostCode;
-
-        }
+    var newVenueDescription = TempData[Dfc.CourseDirectory.WebV2.TempDataKeys.AddedVenueDescription] as string;
```

Comment:
Was there a requirement to keep this notification stuff in there? (I thought this was a pattern we were moving away from?)

Reply:
Yeh these are staying as-is for now.

Reply:
I didn't see a requirement in the story for this - where did it come from? Is this not an opportunity to rip this out rather than creating more tech debt?

Reply:
The notifications were to be unchanged for this story, even if the ACs don't capture it.

Reply:
This seems like a potential gap in refinement to me - is it worth asking the question to PO/UX to get their thoughts?

Reply:
I asked this in the refinement session already.